src/rajoyirize.js

```diff
@@ -61,32 +61,82 @@
         }
 
         function getQuotes() {
-            return quotes || [{ quote: 'no quotes file found'}];
+            var request = new XMLHttpRequest();
+            var quotesSpreadsheetUrl = 'https://spreadsheets.google.com/feeds/list/1tMu-I25eZX1ojpSnB3W4ww6SB9uAbsSKFEwBLmX81F0/od6/public/values?alt=json';

```

Comment:
Is the spreadsheet public? How dangerous is this?

Reply:
We have to find a better solution to store the quotes, but thought that was a good option in the meantime. The spreadsheet is public but not editable.